src/quark2_adapter/utils/config_transformation.py

```diff
@@ -0,0 +1,102 @@
+""" Module to transform on the fly the QUARK 1.0 
```

Comment:
Please add the license header here.

Reply:
done

Reply:
The license header should be at the top, I do not think a doc string for the module is necessary, there is one in the class.